platform/android/java/app/config.gradle

```diff
@@ -8,7 +8,7 @@ ext.versions = [
     fragmentVersion    : '1.3.6',
     nexusPublishVersion: '1.1.0',
     javaVersion        : 11,
-    ndkVersion         : '21.4.7075529' // Also update 'platform/android/detect.py#get_project_ndk_version()' when this is updated.
+    ndkVersion         : '23.2.8568313' // Also update 'platform/android/detect.py#get_project_ndk_version()' when this is updated.
```

Comment:
`get_project_ndk_version` should be replaced with ` get_ndk_version`.

Reply:
Done.